src/build_html/html_builder.js

```diff
@@ -0,0 +1,59 @@
+import Handlebars from "handlebars";
+import fs from "fs";
+import {RoadmapDrawer} from "../scripts/roadmap/roadmap-drawer.js";
+
+export default class HtmlBuilder {
+    constructor(roadmapsJsons){
+        this.roadmapsJsons = roadmapsJsons;
+    };
+    
+    build(){
+        console.log("Creating html:")
+        this.#buildRoadmaps();
+    };
+
+    #buildRoadmaps(){
+        const indexInfo = [];
+        for (let href in this.roadmapsJsons){
```

Comment:
```suggestion
        for (const href in this.roadmapsJsons){
```

href же вроде не меняется. Возможно, меня переклинило, но const можно делать и в циклах.